test/test_files/tck/match/match5.test

```diff
@@ -0,0 +1,463 @@
+n0011
+n0100
+n0101
+n0110
+n0111
+
+# Handling a variable length relationship and a standard relationship in chain, zero length 1
+-LOG Scenario19
+-STATEMENT MATCH (a:A)
+           MATCH (a)-[:LIKES*0]->()-[:LIKES]->(c)
+           RETURN c.name;
+---- 2
+n00
+n01
+
+# Handling a variable length relationship and a standard relationship in chain, zero length 2
+-LOG Scenario20
+-STATEMENT  MATCH (a:A)
+            MATCH (a)-[:LIKES]->()-[:LIKES*0]->(c)
+            RETURN c.name;
+---- 2
+n00
+n01
+
+# Handling a variable length relationship and a standard relationship in chain, single length 1
+-LOG Scenario21
+-STATEMENT MATCH (a:A)
+           MATCH (a)-[:LIKES*1]->()-[:LIKES]->(c)
+           RETURN c.name;
+---- 4
+n000
+n001
+n010
+n011
+
+# Handling a variable length relationship and a standard relationship in chain, single length 2
+-LOG Scenario22
+-STATEMENT MATCH (a:A)
+           MATCH (a)-[:LIKES]->()-[:LIKES*1]->(c)
+           RETURN c.name;
+---- 4
+n000
+n001
+n010
+n011
+
+# Handling a variable length relationship and a standard relationship in chain, longer 1
+-LOG Scenario23
+-STATEMENT MATCH (a:A)
+           MATCH (a)-[:LIKES*2]->()-[:LIKES]->(c)
+           RETURN c.name;
+---- 8
+n0000
+n0001
+n0010
+n0011
+n0100
+n0101
+n0110
+n0111
+
+# Handling a variable length relationship and a standard relationship in chain, longer 2
+-LOG Scenario24
+-STATEMENT MATCH (a:A)
+           MATCH (a)-[:LIKES]->()-[:LIKES*2]->(c)
+           RETURN c.name;
+---- 8
+n0000
+n0001
+n0010
+n0011
+n0100
+n0101
+n0110
+n0111
+
+# Handling a variable length relationship and a standard relationship in chain, longer 3
+-CASE Scenario25
+-STATEMENT  MATCH (d:D)
+            CREATE (e1:E {name: concat(d.name, '0')}),
+                   (e2:E {name: concat(d.name, '1')})
+            CREATE (d)-[:LIKES_D_E]->(e1),
+                   (d)-[:LIKES_D_E]->(e2);
+---- ok
+-STATEMENT MATCH (a:A)
+           MATCH (a)-[:LIKES]->()-[:LIKES*3]->(c)
+           RETURN c.name;
+---- 16
+n00000
+n00001
+n00010
+n00011
+n00100
+n00101
+n00110
+n00111
+n01000
+n01001
+n01010
+n01011
+n01100
+n01101
+n01110
+n01111
+
+# Handling mixed relationship patterns and directions 1
+-CASE Scenario26
+-STATEMENT MATCH (a:A)-[r]->(b:B)
+           DELETE r
+           CREATE (b)-[:LIKES_B_A]->(a);
+---- ok
+-STATEMENT MATCH (d:D)
+           CREATE (e1:E {name: concat(d.name, '0')}),
+                  (e2:E {name: concat(d.name, '1')})
+           CREATE (d)-[:LIKES_D_E]->(e1),
+                  (d)-[:LIKES_D_E]->(e2);
+---- ok
+-STATEMENT MATCH (a:A)
+           MATCH (a)<-[:LIKES]-()-[:LIKES*3]->(c)
+           RETURN c.name;
+---- 16
+n00000
+n00001
+n00010
+n00011
+n00100
+n00101
+n00110
+n00111
+n01000
+n01001
+n01010
+n01011
+n01100
+n01101
+n01110
+n01111
+
+# Handling mixed relationship patterns and directions 2
+-CASE Scenario27
+-SKIP
+-STATEMENT MATCH (a)-[r]->(b)
+           WHERE NOT LABEL(a) = 'A'
+           DELETE r
+           CREATE (b)-[:LIKES]->(a);
```

Comment:
Using the rel table group label `LIKES` doesn't work, but I don't know what to change it to.

Reply:
you can use LIKES_X_Y instead

Reply:
But X and Y each has multiple possibilities, as the labels of `b` and `a` are not fixed, so using any particular possible LIKES_X_Y doesn't work.